src/security/src/main/kotlin/org/vaccineimpact/api/security/SodiumPasswordEncoder.kt

```diff
@@ -0,0 +1,27 @@
+package org.vaccineimpact.api.security
+
+import org.abstractj.kalium.NaCl
+import org.abstractj.kalium.crypto.Password
+import org.abstractj.kalium.encoders.Encoder
+import org.pac4j.core.credentials.password.PasswordEncoder
+
+class SodiumPasswordEncoder : PasswordEncoder
+{
+    private val passwordHelper = Password()
```

Comment:
It would be nice to have a comment here explaining that `Password` uses scrypt with salsa, and that's why it makes sense to be passing in the `SCRYPTSALSA` constants for mem and ops limit. I'm not normally so careful to make things clear, but with this security stuff it's nice to be really clear.